package.json

```diff
@@ -54,22 +59,43 @@
     "babel-plugin-transform-strict-mode": "^6.11.3",
     "babel-preset-es2015": "^6.6.0",
     "babel-register": "^6.7.2",
-    "eslint": "^3.3.1",
+    "eslint": "^4.16.0",
     "eslint-loader": "^1.5.0",
-    "eslint-plugin-babel": "^3.3.0",
-    "webpack": "^3.5.5"
+    "eslint-plugin-babel": "^4.1.2",
+    "webpack": "^3.5.5",
+    "xo": "^0.18.2"
   },
   "dependencies": {
-    "agenda": "^0.9.0",
+    "agenda": "^1.0.3",
```

Comment:
Just noting that we'll need major ver bump because of this.

Reply:
Yeah I may need to change a few things to get it working with the new version. I added @keyvan-m-sadeghi to review as well hoping he may be able to shed some light on how exactly this is all pieced together.

Reply:
Great work @OmgImAlexis, everything looks fine to me, ready to merge?